### PR TITLE
add lock file to 7.0

### PR DESCRIPTION
**Pull request by jsvd**

### CODE REVIEW

File: Gemfile.jruby-2.5.lock.release [hunk 1]
```diff
@@ -0,0 +1,757 @@
+PATH
+  remote: logstash-core-plugin-api
+  specs:
+    logstash-core-plugin-api (2.1.16-java)
+      logstash-core (= 7.0.0.SNAPSHOT)
```
Comment (robbavey) [inline Gemfile.jruby-2.5.lock.release]: Is there an issue with this being `SNAPSHOT`?

Comment (jsvd) [inline Gemfile.jruby-2.5.lock.release]:
```suggestion
      logstash-core (= 7.0.0)
```

File: Gemfile.jruby-2.5.lock.release [hunk 2]
```diff
@@ -0,0 +1,757 @@
...
+
+PATH
+  remote: logstash-core
+  specs:
+    logstash-core (7.0.0.SNAPSHOT-java)
```
Comment (jsvd) [inline Gemfile.jruby-2.5.lock.release]:
```suggestion
    logstash-core (7.0.0-java)
```

**Comment by jsvd** [comment]
well spotted, updated 👍

**Pull request merged**